vcr/filters.py

```diff
@@ -6,6 +6,20 @@
 
 from .util import CaseInsensitiveDict
 
+try:
+    # This supports both brotli & brotlipy packages
+    import brotli
+except ImportError:
+    try:
+        import brotlicffi as brotli
+    except ImportError:
+        brotli = None
+
+
+AVAILABLE_DECOMPRESSORS = {"gzip", "deflate"}
+if brotli is not None:
+    AVAILABLE_DECOMPRESSORS.add("br")
+
```

Comment:
It is easier to expand with future or custom decompressors if `AVAILABLE_DECOMPRESSORS` is a `Dict[[str], Callable[[str], str]]`
```python
AVAILABLE_DECOMPRESSORS = {
    "brotli": brotli.decompress,
    "deflate": zlib.decompress,
    "gzip": lambda body: zlib.decompress(body, zlib.MAX_WBITS | 16),
}
```

Then `decompress_body` can just be
```python
def decompress_body(body, encoding):
    return AVAILABLE_DECOMPRESSORS[encoding](body)
```

Adding a new scheme will be as easy as adding a function to the dict.